pkg/internal/token/testdata/ROPCPoPTokenFromBadPasswordVCR.yaml

```diff
@@ -0,0 +1,247 @@
+---
+version: 2
+interactions:
+    - id: 0
+      request:
+        proto: ""
+        proto_major: 0
+        proto_minor: 0
+        content_length: 0
+        transfer_encoding: []
+        trailer: {}
+        host: ""
+        remote_addr: ""
+        request_uri: ""
+        body: ""
+        form: {}
+        headers:
+            Accept-Encoding:
+                - gzip
+            Client-Request-Id:
+                - e970f474-9424-489a-a3ec-7fa084dccbfc
+            Return-Client-Request-Id:
+                - "false"
+            X-Client-Cpu:
+                - amd64
+            X-Client-Os:
+                - linux
+            X-Client-Sku:
+                - MSAL.Go
+            X-Client-Ver:
+                - 1.2.0
+        url: https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0/.well-known/openid-configuration
```

Comment:
Some data such as tenant id needs to be blanked no ?

Reply:
Redacted tenant ID